src/main/java/net/imglib2/algorithm/gauss3/Gauss3.java

```diff
@@ -116,6 +117,7 @@ public static < S extends NumericType< S >, T extends NumericType< T > > void ga
 	 * {@link Parallelization} context. (By default, it will use the
 	 * {@link ForkJoinPool#commonPool() common ForkJoinPool})
 	 *
+	 * @implNote op name='filter.gauss', type=Computer
```

Comment:
This is the exact Gaussian... need to decide how to name compared to the approximate one.